backend/wmg/data/utils.py

```diff
@@ -97,7 +97,7 @@ def find_dim_option_values(criteria: Dict, snapshot, dimension: str) -> list:
                     # get the set of filters for the specified dimension that are linked to `attr`
                     linked_filter_set = set()
                     for attr in prefixed_attributes:
-                        if dimension in snapshot.filter_relationships[attr]:
+                        if dimension in snapshot.filter_relationships.get(attr, {}):
```

Comment:
This change is to prevent internal service error when the query contains an attribute that doesn't exist in the corpus.